Investigate TestWaitForObjectMissing timing issue

see https://travis-ci.org/deis/builder/builds/113820760:

```console
--- FAIL: TestWaitForObjectMissing (0.01s)
	assert.go:40: /go/src/github.com/deis/builder/pkg/gitreceive/storage/object_test.go:92: the statter wasn't called at least once
```

--- a/pkg/gitreceive/storage/object_test.go
+++ b/pkg/gitreceive/storage/object_test.go
@@ -91,7 +91,11 @@ func TestWaitForObjectMissing(t *testing.T) {
 	err := WaitForObject(statter, bucketName, objKey, 1*time.Millisecond, 2*time.Millisecond)
 	assert.True(t, err != nil, "no error received when there should have been")
 	// it should make 1 call immediately, then calls at 1ms and 2ms
-	assert.True(t, len(statter.Calls) >= 3, "the statter wasn't called at least once")
+	assert.True(
+		t,
+		len(statter.Calls) >= 1,
+		"the statter was not called, but should have been called at least once",
+	)
 }
 
 func TestWaitForObjectExists(t *testing.T) {

